src/api-client.js

```diff
@@ -59,7 +59,9 @@ export default {
   uploadSubmissions(course, labId, formData) {
     return axios.post(`${GRADERX_API}submissions?course=${course}&lab=${labId}&method=file`, formData);
   },
-
+  uploadMossSubmissions(course, labId, clearSubs, useExisting, formData) {
+    return axios.post(`${GRADERX_API}submissions?course=${course}&lab=${labId}${clearSubs? "&clearSubs": ""}${useExisting? "&useExisting": ""}&method=file-moss`, formData);
```

Comment:
I guess there must be a better way to form this URL :sweat_smile:

Reply:
I should've thought that maybe Axios has a neater way to form query params :smile: , solved now but didn't apply it on the others as it will be overridden anyway by PR #49